packages/purpose-process/src/services/purposeService.ts

```diff
@@ -1289,7 +1290,7 @@ export function purposeServiceBuilder(
         consumerDelegation
       );
 
-      if (purposeIsDraft(purposeToClone.data)) {
+      if (isClonable(purposeToClone.data)) {
         throw purposeCannotBeCloned(purposeId);
       }
```

Comment:
This code is contradictory: "If purpose is clonable --> then purpose cannot be cloned"
The "isClonable" validator should check all the states **except** Draft and Archived, then this code can be refactored negating the condition 👇 
```suggestion
      if (!isClonable(purposeToClone.data)) {
        throw purposeCannotBeCloned(purposeId);
      }
```

Reply:
My bad, you're right